Libraries/Text/TextInput/RCTBackedTextInputDelegateAdapter.m

```diff
@@ -382,7 +382,9 @@ - (void)textViewDidChangeSelection:(__unused UITextView *)textView
 {
   if (_lastStringStateWasUpdatedWith && ![_lastStringStateWasUpdatedWith isEqual:_backedTextInputView.attributedText]) {
     [self textViewDidChange:_backedTextInputView];
-    _ignoreNextTextInputCall = YES;
+
+    if (![_backedTextInputView isGhostTextChanging]) // [macOS]
+      _ignoreNextTextInputCall = YES;
```

Comment:
nit: Can we add brackets? I'm not a fan of single line if() statements that don't use brackets..